app/views/provider_interface/application_choices/notes.html.erb

```diff
@@ -3,9 +3,6 @@
 <% content_for :before_content do %>
   <div class="govuk-breadcrumbs">
     <ol class="govuk-breadcrumbs__list">
-      <li class="govuk-breadcrumbs__list-item">
-        <%= link_to 'Applications', provider_interface_applications_path, class: 'govuk-breadcrumbs__link' %>
-      </li>
       <li class="govuk-breadcrumbs__list-item">
```

Comment:
I don't think preserving the candidate's name as a breadcrumb provides any value now that we have the tabbed navigation. I'd probably remove the entire breadcrumbs section from all three views.